src/views/home/sections/Landing.tsx

```diff
@@ -25,11 +25,11 @@ function Landing() {
 		<section className={styles.homeLanding}>
 			<TitleBanner className={styles.homeBanner}>
 				<div className="d-flex flex-column-reverse">
-					<h1>HackUCI</h1>
+					<h1>Hack at UCI</h1>
 					<span className={styles.landingDate}>February 3 &ndash; 5, 2023</span>
 				</div>
 
-				<Row className={styles.homeButtons}>
+				<Row className={styles.homeButtons + " " + "justify-content-center"}>
```

Comment:
Question: why did we need to include this additional style?

Reply:
To be honest, I don't remember. I think it was because the buttons were off center when I was working on it but now I'm not noticing it. I'll remove it.